.idea/codeStyles/Project.xml

```diff
@@ -1,125 +0,0 @@
-<component name="ProjectCodeStyleConfiguration">
```

Comment:
If you think this file should not be tracked, please edit `.gitignore`
(But is there a reason to untrack this file? )